config/group_vars/k8s-cluster/k8s-cluster.yml

```diff
@@ -77,12 +87,12 @@ kube_network_plugin: calico
 kube_network_plugin_multus: false
 
 # Kubernetes internal network for services, unused block of space.
-kube_service_addresses: 10.96.0.0/12
+kube_service_addresses: 10.233.0.0/18
 
 # internal network. When used, it will assign IP
 # addresses from this range to individual pods.
 # This network must be unused in your network infrastructure!
-kube_pods_subnet: 192.168.0.0/16
+kube_pods_subnet: 10.233.64.0/18
```

Comment:
It probably doesn't matter much but these are now different from what we have in ck8s-cluster. In any case I guess any terraform code written to work with kubespray will probably have defaults that work with the default kubespray values rather than default ck8s-cluster values so I think this change is fine. :slightly_smiling_face:

Reply:
Yea, I thought so as well. And these ranges shouldn't matter for anyone anyway since we should use DNS for everything anyway. So it might be a good test to see if apps works with this :)